src/frontend/ts/state/chessStore.ts

```diff
@@ -14,33 +14,45 @@ const useChessStore = create<ChessState>((set, get) => ({
   setUrbit: (urbit: Urbit) => set({ urbit }),
   setDisplayGame: (displayGame: ActiveGameInfo | null) => set({ displayGame }),
   setPracticeBoard: (practiceBoard: String | null) => set({ practiceBoard }),
-  receiveChallenge: (data: ChallengeUpdate) =>
-    {
-      switch (data.chessUpdate) {
-        case Update.ChallengeSent: {
-          set(state => ({ outgoingChallenges: state.outgoingChallenges.set(data.who, data as ChallengeSentUpdate) }))
-          break
-        }
-        case Update.ChallengeReceived: {
-          set(state => ({ incomingChallenges: state.incomingChallenges.set(data.who, data as ChallengeReceivedUpdate) }))
-          break
-        }
-        case Update.ChallengeDeclined: {
-          get().removeOutgoingChallenge(data.who)
-          break
-        }
-        case Update.ChallengeResolved: {
-          get().incomingChallenges.delete(data.who)
-          break
-        }
-        default: {
-          console.log('RECEIVED BAD UPDATE')
-          console.log(data.chessUpdate)
-          console.log((data as ChallengeUpdate).chessUpdate)
-          console.log((data as ChallengeUpdate).who)
+  receiveChallengeUpdate: (data: ChallengeUpdate) => {
+    switch (data.chessUpdate) {
+      case Update.ChallengeSent: {
+        set(state => ({ outgoingChallenges: state.outgoingChallenges.set(data.who, data as ChallengeSentUpdate) }))
+        break
+      }
+      case Update.ChallengeReceived: {
+        set(state => ({ incomingChallenges: state.incomingChallenges.set(data.who, data as ChallengeReceivedUpdate) }))
+        break
+      }
+      case Update.ChallengeDeclined: {
```

Comment:
See primary comment for this review. This should be renamed to `ChallengeResolved`.